src/main/java/heronarts/lx/pattern/color/SolidPattern.java

```diff
@@ -33,6 +33,8 @@
 
   public SolidPattern(LX lx) {
     this(lx, LXColor.RED);
+    this.color.mode.setValue(LinkedColorParameter.Mode.PALETTE);
```

Comment:
Let's hold off on this default-change for now, I think for most non-power users with stock LX sticking with a basic direct color as default is best.

If this is driving you bonkers having to change all the time, you can make your own subclass of SolidColorPattern for your project which flips these defaults at construction-time and register that in your initialize(), same as Alternating

Reply:
Done